app/src/components/adminConsole/publishedCups/PublishedCups.tsx

```diff
@@ -10,8 +10,7 @@ const PublishedCups: React.FunctionComponent = (props: {
 }) => {
     return (
         <div className="PublishedCups">
-            <PublishedCupsHeader />
-            <SectionHeaderUnderline />
+            <PublishedCupsHeader title={"Published Cups"} />
```

Comment:
You could just use `SectionHeading` directly here coudn't you?

Reply:
yup, done